SonarQube.Bootstrapper/BootstrapperSettings.cs

```diff
@@ -65,9 +65,6 @@ public class BootstrapperSettings : IBootstrapperSettings
         #endregion
 
         private readonly ILogger logger;
-        private readonly string sonarQubeUrl;
```

Comment:
Auto-properties work, but don't capture the intent that the values are immutable (I think they were auto-properties before and I changed them to readonly fields).